src/Chapter20/Listing20.02.AsynchronousWebRequest.cs

```diff
@@ -25,15 +26,16 @@ public static void Main(string[] args)
                 url = args[1];
                 // Ignore additional parameters
             }
-            Console.Write($"Searching for '{findText}' at URL '{url}'.");
+            Console.WriteLine(
+                $"Searching for '{findText}' at URL '{url}'.");
 
             using WebClient webClient = new WebClient();
-            Console.Write("\nDownloading...");
+            Console.Write("Downloading...");
             Task task = webClient.DownloadDataTaskAsync(url)
                 .ContinueWith(antecedent =>
                 {
                     byte[] downloadData = antecedent.Result;
-                    Console.Write("\nSearching...");
+                    Console.Write($"{Environment.NewLine}Searching...");
```

Comment:
```suggestion
                    Console.WriteLine("Searching...");
```
Use writeline instead of a character return line feed